### PR TITLE
Feature: Added pushToCardPayment

### PR DESCRIPTION
**Pull request by kirill-moves**
Added support for [PushToCardPayment](https://docs.unit.co/push-to-card-payments/)

### CODE REVIEW

**Review by dev-dennis-guye — COMMENTED** [review]
Can you revert the linter changes?

File: types/common.ts [hunk 1]
```diff
@@ -425,124 +425,139 @@ export type Beneficiary = Pick<BaseContactAttributes, "fullName" | "dateOfBirth"
...
+  details?: string
+  meta?: { supportId?: string; }
+  source?: { pointer: string; }
+  [k: string]: unknown // allow for other keys
+}
+
+/**
+ * https://docs.unit.co/types/#push-to-card-configuration
+ */
+export interface Configuration {
```
Comment (dev-dennis-guye) [inline types/common.ts]: Why did you choose to add this in Common?

Comment (dev-dennis-guye) [inline types/common.ts]:
```suggestion
export interface PushToCardConfiguration {
```

Comment (kirill-moves) [reply types/common.ts]: Followed suit of where other attributes exist, happy to move this somewhere else

Comment (dev-dennis-guye) [reply types/common.ts]: Ok, your reasoning makes sense. Could you also move this type above UnitError?

**Review by dev-dennis-guye — APPROVED** [review]
One qq: 

@cassiano-moves do you recall if Unit rolled our a new event for push to card? I can't find it in their docs but I recall that we spoke about it during our calls.

**Comment by dev-dennis-guye** [comment]
Also you need to rebase your work on fork.

**Comment by cassiano-moves** [comment]
> One qq:
> 
> @cassiano-moves do you recall if Unit rolled our a new event for push to card? I can't find it in their docs but I recall that we spoke about it during our calls.

@Dennis, I'm not 100% sure if it's been rolled out, but in the docs, it's in the first paragraph here - https://docs.unit.co/push-to-card-payments/ and within each event it now mentions PushToCardPayment as a type. This is the thread in which Unit confirmed the event - [thread](https://gonuco.slack.com/archives/C01KX2895GW/p1684327888439799?thread_ts=1683067613.191119&cid=C01KX2895GW)

**Comment by dev-dennis-guye** [comment]
> > One qq:
> > @cassiano-moves do you recall if Unit rolled our a new event for push to card? I can't find it in their docs but I recall that we spoke about it during our calls.
> 
> @dennis, I'm not 100% sure if it's been rolled out, but in the docs, it's in the first paragraph here - https://docs.unit.co/push-to-card-payments/ and within each event it now mentions PushToCardPayment as a type. This is the thread in which Unit confirmed the event - [thread](https://gonuco.slack.com/archives/C01KX2895GW/p1684327888439799?thread_ts=1683067613.191119&cid=C01KX2895GW)

Ok @kirill-moves  you are good to go. Unit is reusing an existing event.

**Pull request merged**